batch/batch/front_end/query/query.py

```diff
@@ -506,78 +506,78 @@ def query(self) -> Tuple[str, List[str]]:
         if isinstance(self.operator, PartialMatchOperator):
             value = f'%{value}%'
         sql = f"""
-((batches.id) IN
- (SELECT batch_id FROM job_group_attributes
+((job_groups.batch_id, job_groups.job_group_id) IN
```

Comment:
I decided to make this a separate issue to fix all of these queries and optimize them. It's a much larger project than what I want to tackle here.